src/hume/empathic_voice/chat/audio/microphone_interface.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from typing import ClassVar
+
+from hume.empathic_voice.chat.audio.microphone import _Microphone
+from hume.empathic_voice.chat.audio.microphone_sender import _MicrophoneSender
+from hume.empathic_voice.chat.socket_client import ChatWebsocketConnection
+from hume.empathic_voice.chat.audio.chat_client import _ChatClient
```

Comment:
Just an FYI, typically if a module is private you can use it within a package, but not outside a package. But if a module member (e.g. ChatClient) is private you should not use it outside that module. So to keep things internal to a package I try to use private modules.

Reply:
Got it, thank you for the clarification - I made the change to remove the underscores from classes and will keep this in mind for future updates, both to the SDK and example code. 📝 💫